src/components/AddressTagsPage.tsx

```diff
@@ -52,8 +42,19 @@ const AddressTagsPage = ({
             setRetryFunction(fetchRecords);
           }
         });
+      if (res) {
```

Comment:
When would there not be `res`? Also you should check for `res.records` and other stuff, no?

Reply:
`res` should always exist. You're right.